frontend/src/components/filters.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  export let movieChecked = true
+  export let tvChecked = true
+  export let search
+
+  // If the cookie should break and set both to false
+  $: {
+    if (!movieChecked && !tvChecked) {
+      movieChecked = true
+      tvChecked = true
+    }
+  }
+</script>
+
+<div class="dropdown dropdown-end">
```

Comment:
It would be nice if pressing the dropdown again would close it - This is especially annoying on phone

Here's an implementation idea: https://github.com/saadeghi/daisyui/issues/157#issuecomment-1074349200
And on the REPL: https://svelte.dev/repl/4c5dfd34cc634774bd242725f0fc2dab?version=3.46.4

Reply:
Can this happen in a new PR and ill fix all 3 buttons

Reply:
Fine :D Approving when there's an issue <3

Reply:
#265